### PR TITLE
fix(docs): add new extensions to community extensions page

### PR DESCRIPTION
**Pull request by dev-hitesh-gupta**
updated extension page with the dynamic data source component which helps in connecting loopback with databases runtime.

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Community-extensions.md [hunk 1]
```diff
@@ -71,6 +71,10 @@ list.
       <td><a href="https://www.npmjs.com/package/loopback-rabbitmq">loopback-rabbitmq</a></td>
       <td>An Rabbitmq extension that implements Queue consumers and producers</td>
     </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback4-dynamic-datasource">loopback4-dynamic-datasource</a></td>
+      <td>To connect data sources dynamically major use case is when using loopback for the multi-tenant physical separated databases</td>
```
Comment (dhmlau) [inline docs/site/Community-extensions.md:76]: To be consistent with the description of other extensions in this doc, would it work if we reword it as:

Connect data sources dynamically during runtime

**Comment by dhmlau** [comment]
@dev-hitesh-gupta, thanks for adding the community extension to the list. Please review my comment https://github.com/loopbackio/loopback-next/pull/7674/files#r676931931. Thanks.

**Comment by achrinza** [comment]
Thanks for adding the LoopBack extension!

Please also squash the PR into a single commit. A suggested commit message would be:

```
feat(docs): add loopback4-dynamic-datasource community extension
```

File: docs/site/Community-extensions.md (hunk 1 above)
Comment (dev-hitesh-gupta) [reply docs/site/Community-extensions.md:76]: yes

Comment (dev-hitesh-gupta) [reply docs/site/Community-extensions.md]: @dhmlau Done with changes you can review it again Sorry for the delay.

**Comment by dev-hitesh-gupta** [comment]
> Thanks for adding the LoopBack extension!
> 
> Please also squash the PR into a single commit. A suggested commit message would be:
> 
> ```
> feat(docs): add loopback4-dynamic-datasource community extension
> ```

Done

**Comment by dev-hitesh-gupta** [comment]
> @dev-hitesh-gupta, thanks for adding the community extension to the list. Please review my comment https://github.com/loopbackio/loopback-next/pull/7674/files#r676931931. Thanks.

Done

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
@dev-hitesh-gupta, thanks for working through this. Your PR has landed!

**Pull request merged**